scripts/ceph-windows/setup_ceph_vstart

```diff
@@ -43,6 +45,14 @@ time rsync_cmd $WORKSPACE/ceph ${VSTART_DIR}/build-ceph-vstart.sh ${UBUNTU_SSH_U
 time SSH_TIMEOUT=1h ssh_exec ./build-ceph-vstart.sh
 ssh_exec sudo apt-get install -y python3-prettytable
 
+memstore_args=""
+bluestore_args=""
+if [ $USE_MEMSTORE == "yes" ]; then
+    memstore_args="--memstore -o memstore_device_bytes=$VSTART_MEMSTORE_BYTES"
+else
+    bluestore_args="-o bluestore_block_size=$VSTART_BLUESTORE_BYTES"
```

Comment:
I'd make this a single `objectstore_args` variable and pass `--bluestore` here for consistency.

Reply:
Updated, thanks for the suggestion.